kimchi/src/proof.rs

```diff
@@ -107,7 +116,7 @@ pub struct ProverProof<G: AffineCurve> {
 
     /// Two evaluations over a number of committed polynomials
     // TODO(mimoo): that really should be a type Evals { z: PE, zw: PE }
-    pub evals: [ProofEvaluations<Vec<G::ScalarField>>; 2],
+    pub evals: [ProofEvaluations<G::ScalarField>; 2],
```

Comment:
Will it still be relevant to have the `ConsecutiveEvals` type? [PR 607](https://github.com/o1-labs/proof-systems/pull/607), I would have to adapt it to the current master, but it should be easy